ekf/tests/test_runner/main.c

```diff
@@ -0,0 +1,125 @@
+/*
+ * Phoenix-Pilot
+ *
+ * EKF test runner
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Piotr Nieciecki
```

Comment:
Please elaborate a little here (one/two sentences):

```suggestion
 * EKF test runner
 *
 * Executes ekf (...) via devekf (...) via copied `ekf.conf` file
 *
 * Copyright 2023 Phoenix Systems
 * Author: Piotr Nieciecki
```